### ISSUE TITLE
How to open source chapter

### ISSUE THREAD

**Issue by evelinag** [alan-turing-institute/the-turing-way#685]
## Detailed description
 * This issue is a place to discuss matters relating to the writing of the chapter on "How to do open source"
 * This chapter is being written in _pull_request_number_
 * This chapter is being developed on the _chapter_topic_ branch.

### Table of contents

An extremely rought list of things that should go into the chapter:

- creating an open source project
- how to choose a license
- contributing to an already existing open source project
- incentives for open source
- pros and cons of getting involved, work-life balance
- how to make an open source project accessible to new contributors
- community management
- contributing guidelines
- code of conduct

### Resources

There will be an acompanying repo with practical exercies and slides.

## Current status
This chapter is empty. If anyone would like to make a start they are more than welcome to do so.

 - [ ] Write chapter outline
 - [ ] Add material to the chapter
 - [ ] Combine materials into a readable chapter
 - [ ] Proofread
 - [ ] Request reviews
 - [ ] Address reviews
 - [ ] Merge to master branch.

**Comment by KirstieJane** [comment]
YES! This does sound like a whole volume! But I wildly love all of the content here!

Lets brainstorm at the October sprint about how we can split up the content to be most manageable (to read and to write)

Thank you though - so so amazing!

**Comment by sgibson91** [comment]
@KirstieJane, @malvikasharan and I had a discussion over lunch about what an REG contribution on this topic might look like during our Open Source Dash on 29/11/2019. Rough notes captured here: https://hackmd.io/6wAL7XwcQc6-EFohOWXOhw?view

**Comment by sgibson91** [comment]
Found a blog post as well https://www.twilio.com/blog/learn-contribute-open-source

**Comment by sgibson91** [comment]
Some parts of the [Open Research chapter](https://the-turing-way.netlify.com/open_research/open_research.html) may be related/cross-referenced for this chapter skeleton.

Chapter skeleton to go into Collaboration volume.

**cc:** @LouiseABowler

**Comment by JimMadge** [comment]
Proposal for unravelling the [Open Research chapter](https://the-turing-way.netlify.com/open_research/open_research.html),

Rephrase the [Open Source Software](https://the-turing-way.netlify.com/open_research/02/opensourcesoftware.html) section to focus on the research benefits open source and how to make your research software open source.

Take the content about the _personal_ benefits of contributing to open source projects and advice for how to contribute and place it in the new chapter.

Roughly this would correspond to,
Subsections to remain:
- What is open source software?
- How open source software benefits research
- How to run your own open source software project
- Closed software

Subsections to transfer:
- How running and contributing to open source software projects benefits you
- How to contribute to other’s open source software projects

This topics discussed in the later subsections are very similar to those brought up in the [open source dash](https://hackmd.io/Hm4rI2hUSUCocRriKkmUDA?both)

**Comment by radka-j** [comment]
A number of the topics suggested here are already covered (at least to an extent) in the [Open source software](https://the-turing-way.netlify.com/open_research/02/opensourcesoftware.html) section of the Open Research chapter.

One idea would be to take this entire section, move it to a new volume/chapter  and split it up into chapters/sections:
- What is open source software
- Pros/cons of open source
- Contributing to existing open source projects
   - What's a useful contribution
   - What to look for in an open source project before contributing
   - How to contribute
- Managing your own open source project

Each topic could then be further developed. For example, "cons" are not discussed in the chapter at all.

**Comment by EstherPlomp** [comment]
Hello from 2023 @evelinag, @JimMadge, @radka-j! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. 

It looks like this issue is no longer being actioned, so we'll be closing it for now - however, please feel free to re-open it to continue working on it at any point! If you have any questions or concerns, please comment below ✨ Thank you!
